docs/_docs/04_reference/cli-options.md

```diff
@@ -762,5 +767,65 @@ block by specifying `<BLOCK>.<ATTR>`, where `<BLOCK>` is the block name: e.g., `
 
 **CLI Arg**: `--terragrunt-json-out`
 **Requires an argument**: `--terragrunt-json-out /path/to/terragrunt_rendered.json`
+**Commands**:
+- [render-json](#render-json)
 
 When passed in, render the json representation in this file.
+
+
+### terragrunt-modules-that-include
+
+**CLI Arg**: `--terragrunt-modules-that-include`
+**Requires an argument**: `--terragrunt-modules-that-include /path/to/included-terragrunt.hcl`
+**Commands**:
+- [run-all](#run-all)
+- [plan-all (DEPRECATED: use run-all)](#plan-all-deprecated-use-run-all)
+- [apply-all (DEPRECATED: use run-all)](#apply-all-deprecated-use-run-all)
+- [output-all (DEPRECATED: use run-all)](#output-all-deprecated-use-run-all)
+- [destroy-all (DEPRECATED: use run-all)](#destroy-all-deprecated-use-run-all)
+- [validate-all (DEPRECATED: use run-all)](#validate-all-deprecated-use-run-all)
+
+When passed in, `run-all` will only run the command against Terragrunt modules that include the specified file.
+
+This applies to the set of modules that are identified based on all the existing criteria for deciding which modules to
+include. For example, consider the following folder structure:
```

Comment:
NIT: You show the file structure here, but not the contents of the files, so it's not obvious what uses `include` on what if you're not already familiar with Terragrunt.

Reply:
Makes sense! Added in example: 8eb213b4567d56ed6f2fdd8dfb807414a280a2ae